editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1262,9 +1262,7 @@ void AnimationPlayerEditor::_seek_value_changed(float p_value, bool p_set, bool
 
 	if (!p_timeline_only) {
 		if (player->is_valid() && !p_set) {
-			double delta = player->get_current_animation_position();
 			player->seek(pos, true, true);
-			player->seek(pos + delta, true, true);
```

Comment:
This particular line of code caused the animation display to jump elsewhere of the actual `pos`.

Reply:
This change is equivalent to the following:
```cpp
	if (!p_timeline_only) {
		if (player->is_valid() && !p_set) {
			double delta = player->get_current_animation_position();
			player->seek(pos, true, true);
			player->seek(pos + delta, true, true);
		} else {
			if (player->is_playing()) {
				player->stop();
			}
			player->seek(pos, true, true);
		}
	}
```
↓
```cpp
	if (!p_timeline_only) {
		player->seek(pos, true, true);
	}
```
I can't recall anymore why it was necessary in the past to handle drags and seek by key separately, but I think this change probably fine (you just need to rewrite the code as suggested). @SaracenOne Do you have any idea?